internal/manifests/manifestutils/paths.go

```diff
@@ -14,6 +14,16 @@ const (
 	// ReceiverTLSCertDir returns the mount path of the receivers certificates (for ingesting traces).
 	ReceiverTLSCertDir = TLSDir + "/receiver"
 
+	// ReceiverGRPCTLSCADir is the path that is mounted from the configmap for TLS for receiver.
+	ReceiverGRPCTLSCADir = "/var/run/ca-receiver/grpc"
+	// ReceiverGRPCTLSCertDir returns the mount path of the receivers certificates (for ingesting traces).
+	ReceiverGRPCTLSCertDir = TLSDir + "/receiver/grpc"
+
+	// ReceiverHTTPTLSCADir is the path that is mounted from the configmap for TLS for receiver.
+	ReceiverHTTPTLSCADir = "/var/run/ca-receiver/http"
+	// ReceiverHTTPTLSCertDir returns the mount path of the receivers certificates (for ingesting traces).
+	ReceiverHTTPTLSCertDir = TLSDir + "/receiver/http"
```

Comment:
Wht don't we use `path.Join` here?

Reply:
We can but I don't think we need it, this are constants, we don't need to deal with double slashes, empty strings etc..